redhat/firehol/firehol.spec

```
@@ -146,22 +148,27 @@ fi
 %endif
 %{_sbindir}/firehol
 %{_sbindir}/fireqos
+%{_sbindir}/update-ipsets
 %{_docdir}/firehol/html/*
 %{_docdir}/firehol/contrib/*
 %{_docdir}/firehol/examples/*
 %{_docdir}/firehol/*.pdf
 %{_mandir}/man1/*.1*
 %{_mandir}/man5/*.5*
+%{_datadir}/update-ipsets/webdir/*
 %dir %{_sysconfdir}/firehol/services/
 %{_localstatedir}/spool/firehol
 %{_libexecdir}/firehol/%{version}/firehol
 %{_libexecdir}/firehol/%{version}/fireqos
+%{_libexecdir}/firehol/%{version}/update-ipsets
 %{_libexecdir}/firehol/%{version}/functions.common
 %{_libexecdir}/firehol/%{version}/install.config
 %{_libexecdir}/firehol/%{version}/services.common
 %{_libexecdir}/firehol/%{version}/services.firehol
 %{_libexecdir}/firehol/%{version}/services.fireqos
 
 %changelog
+* Sat Feb 15 2020 John Ramsden <johnramsden@riseup.net>
```

Include the updated Release number here

```suggestion
* Sat Feb 15 2020 John Ramsden <johnramsden@riseup.net> - 3.1.1-2
```